js/tools/rainbowBeads.js

```diff
@@ -0,0 +1,47 @@
+/* global app */
+window.tools.rainbow = {
+  name: 'rainbow',
```

Comment:
@andrewscottcohen don't forget that ur tool's object name `window.tools.rainbow` should match the file name (in camel case), like it says in the class note: 

> Replace EXAMPLE with the name of your tool, your JavaScript file's name should match that of your tool. This name should be written in lower case and not contain any spaces (use [camel case](https://en.wikipedia.org/wiki/Camel_case) for names containing more than one word).